docs(dpg): sample convenience method `GetCrops`

archerzz opened the pull request:
related to https://github.com/Azure/autorest.csharp/issues/2199

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

archerzz committed:
docs(dpg): sample convenience method `GetCrops`

related to https://github.com/Azure/autorest.csharp/issues/2199

azure-sdk commented:
**API change check for `Azure.Verticals.AgriFood.Farming`**

API changes have been detected in `Azure.Verticals.AgriFood.Farming`. You can review API changes [here](https://apiview.dev/Assemblies/Review/975e922a55ee4280939bd5e64b711bb3?diffOnly=True&diffRevisionId=02321a84773a4a2085a7a1ca3bd40bf1)

archerzz committed:
replace multiple optional parameters with an Option parameter

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.Verticals.AgriFood.Farming](https://apiview.dev/Assemblies/Review/975e922a55ee4280939bd5e64b711bb3)

archerzz committed:
remove unnecessary `Value` suffix to convenience method

1. make `GetCropsOptions` as a required parameter
2. remove `Value` suffix from `GetCrops` methods

archerzz commented:
close since board review is complted
